pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "oldest-supported-numpy",
+    "numpy>=2.0.0rc2,<3",
```

Comment:
Is this a problem? Should we reduce this requirement, what if someone is using 1.21.6 wants to install mayavi?

Reply:
This is for the build system. So if someone does `pip install .` or whatever, it should do an isolated build using NumPy 2+. This should be fine for them since wheels compiled with NumPy 2 are backward compatible with NumPy 1.x.

Reply:
Yes, but since numpy 2.x does not work with 3.8, those on an older Python cannot use Mayavi. Do we want that or is Python 3.8 ancient history?

Reply:
Even though security support for 3.8 isn't over yet (it ends in 5 months), NumPy made the decision to drop it already. So to me it makes sense for Mayavi to drop it as well.

If it were trivial to support 3.8 I'd say "sure let's keep it", but I'm not sure how hard it would be to fix the errors seen here for example:

https://github.com/enthought/mayavi/actions/runs/9273915257/job/25515059891

```
  tvtk/src/array_ext.c:3467:13: error: call to undeclared function 'PyArray_MultiIter_ITERS'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
```

I tried for a little while but it wasn't trivial for me at least.

Reply:
Thanks for the clarification, in any case people could use the previous release of mayavi if they really want to use Python 3.8.